README.md

```diff
@@ -1 +1,16 @@
-# hw01
\ No newline at end of file
+# hw01
+
+# Biography
+### Heather Weller
```

Comment:
Typically you want to include a space after headers because sometimes certain devices will not render this properly. This wasn't the case here, but it's just something to consider.

Reply:
Thanks for the help, is this the reason I got marked down on the reproducibility part? Also thanks for the comments about committing, now that I understand how it works better, I will do that moving forward.